module_utils/network/junos/config/lacp_interfaces/lacp_interfaces.py

```diff
@@ -0,0 +1,247 @@
+#
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""
+The junos_lacp_interfaces class
+It is in this file where the current configuration (as dict)
+is compared to the provided configuration (as dict) and the command set
+necessary to bring the current configuration to it's desired end-state is
+created
+"""
+
+from ansible.module_utils.network.common.cfg.base import ConfigBase
+from ansible.module_utils.network.common.utils import to_list, get_xml_conf_arg
+from ansible.module_utils.network.junos.facts.facts import Facts
+from ansible.module_utils.network.junos.junos import locked_config, load_config, commit_configuration, discard_changes, tostring
+from ansible.module_utils.network.junos.utils import utils as junosutils
+
+import q
```

Comment:
I guess this is not inteded to be imported in the final revision ;)

Reply:
Besides that, we should deff agree and merge on tox.ini flake8 validations for this repo. IMHO we should just match ansible/ansible for now